src/Components/LoginForm.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
```

Comment:
Minor detail, but this component does not need to be a class component. Could we make it into a functional component as to avoid using class components when they are not needed?

Reply:
Also, are we happy with the styling of this component. I.e. could we perhaps add a button UI component?